dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/utils/DataRequestsFilter.kt

```diff
@@ -0,0 +1,83 @@
+package org.dataland.datalandcommunitymanager.utils
+import org.dataland.datalandbackend.openApiClient.model.DataTypeEnum
+import org.dataland.datalandbackendutils.model.KeycloakUserInfo
+import org.dataland.datalandcommunitymanager.model.dataRequest.AccessStatus
+import org.dataland.datalandcommunitymanager.model.dataRequest.RequestStatus
+import org.dataland.datalandcommunitymanager.services.KeycloakUserControllerApiService
+
+/**
+ * A filter class used in the searchDataRequestEntity-Method which allows
+ * convenient usage of SEPL instructions in the query
+ */
+data class DataRequestsFilter(
+    val dataType: Set<DataTypeEnum>?,
+    val userId: String?,
+    val emailAddress: String?,
+    val datalandCompanyId: String?,
+    val reportingPeriod: String?,
+    val requestStatus: Set<RequestStatus>?,
+    val accessStatus: Set<AccessStatus>?,
+) {
+    val shouldFilterByDataType: Boolean
+        get() = dataType?.isNotEmpty() ?: false
+
+    val usedDataTypeFilter: List<String>
```

Comment:
Would remove the "used" prefix

Reply:
From my point of view it doesn't add much value to the variable in terms of clarity. But we can have a call to discuss this.